### DISCUSSION TITLE
How to list all apps and versions

### DISCUSSION THREAD

**Discussion by oss-linux-etc** [dat-linux/community#3] [General]
Is there a way to list all apps and their version whether installed or not?? thx

**Comment by dat-linux** [comment]
Hi, run this in a terminal:

cari plugins | grep "https" | cut -d " " -f1 | xargs -I{} sh -c 'cari add $1 > /dev/null 2>&1; v="\`cari latest $1\`"; echo $1 $v' sh {}

**Reply by jichak901** [reply]
Hi,
the control panel App Updater is not working for a few days,
because gitlab asks for a username. Can you help me? thx

**Reply by dat-linux** [reply]
Hi, can you please post a screenshot?

**Reply by dat-linux** [reply]
Try running the following on the command line (false/archived packages):
```
cari remove base
cari remove lab_plot
cari remove unipept
cari remove unipept-cli
cari remove avogadro
cari update
```

and see if that resolves things. If not manually remove any of those 5 false packages from $HOME/.cari/respository/ dir, and also $HOME/.cari/plugins/ if necessary.

**Reply by dat-linux** [reply]
also run: 

`sudo apt-get update && sudo apt-get upgrade datlinux-ctrl-pnl`

**Comment by jichak901** [comment]
Hi,
thank you for your advice! Running this commands resolve the error:

cari remove base
cari remove unipept
cari remove unipept-cli
cari remove avogadro
cari update

So sorry but I didn't take a screenshot before running the commands


















DAT Linux ***@***.***> ezt írta (időpont: 2022. júl. 29., P,
23:10):
